.circleci/config.yml

```diff
@@ -1,14 +1,76 @@
-# Golang CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-go/ for more details
 version: 2
 jobs:
-  build:
-    docker: # use the docker executor type; machine and macos executors are also supported
-      - image: circleci/golang:1.9
+  sanity-check:
+    working_directory: /go/src/github.com/ligato/networkservicemesh
+    docker:
+      - image: circleci/golang:1.10
+    steps:
+      - checkout
+      - run: sudo apt-get install yamllint shellcheck python3-pkg-resources
+      - run: yamllint -c .yamllint.yml $(git ls-files '*.yaml' '*.yml' | grep -v 'vendor/')
+      - run: make check dep-check
 
+  build:
     working_directory: /go/src/github.com/ligato/networkservicemesh
+    docker:
+      - image: circleci/golang:1.10
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: go install ./vendor/k8s.io/kube-openapi/cmd/openapi-gen
+      - run: make verify docker-build
 
+  packet-deploy:
```

Comment:
I looked at the circle-ci dashboard, and it might be good to dump some information on the Kubernetes cluster after it's deployed. Since we do not have access to the logs after this run is done, the more information we can save here the easier it will be to debug these. For example, the items we collect after the integration tests [here](https://github.com/ligato/networkservicemesh/blob/master/scripts/integration-tests.sh#L138) would be useful. And it's worth collecting that information after deploy in case the deploy itself fails somehow.

I'm fine with an issue being opened and this being fixed after this merges however.

Reply:
Yeah, dumping K8s logs to CircleCI, particular on failure is going to be huge for debuggability :)

Reply:
circle-ci run same integration-tests.sh script and will collect same logs after integration tests run. Going to add some log collection on deploy phase as well.